wfe/web-front-end.go

```diff
@@ -549,66 +564,38 @@ func (wfe *WebFrontEndImpl) Registration(response http.ResponseWriter, request *
 	} else if id <= 0 {
```

Comment:
I know this isn't in this review, but I should go add a util method to obtain the Int64 value of the ID from a idStr that we can use for all these places.